sockeye/layers.py

```diff
@@ -325,9 +293,9 @@ def cast(self, dtype):
         self._dtype = dtype
         super().cast(dtype)
 
-    def hybrid_forward(self, F, queries, keys, values, lengths=None, bias=None):
-        # (n, lq, lk)
-        logits = F.batch_dot(lhs=queries, rhs=keys, transpose_b=True)
+    def hybrid_forward(self, F, queries, key_values, heads, lengths=None, bias=None):
```

Comment:
Update calls/docstrings at `ProjectedDotAttention` (https://github.com/awslabs/sockeye/pull/884/files#diff-74c804465509cbab074609c50c13a8f7L710) and `PlainDotAttention` (https://github.com/awslabs/sockeye/pull/884/files#diff-74c804465509cbab074609c50c13a8f7L642).